test/test_routes.py

```diff
@@ -400,12 +400,15 @@ def test_mount_gradio_app_raises_error_if_event_queued_but_queue_disabled(self):
 
     def test_proxy_does_not_leak_hf_token_externally(self):
         gr.context.Context.hf_token = "abcdef"
-        r = routes.App.build_proxy_request(
-            "https://gradio-tests-test-loading-examples-private.hf.space/file=Bunny.obj"
-        )
-        assert "authorization" in dict(r.headers)
-        r = routes.App.build_proxy_request("https://google.com")
-        assert "authorization" not in dict(r.headers)
```

Comment:
Did you mean to delete this test? Auth headers are orthogonal to URL restriction.